tests/comments/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -626,7 +630,9 @@ do {
     // some comment
 } while (true);
 
-$test = foo ?: bar(function () {
+$test =
+    foo
+    ?: bar(function () {
```

Comment:
Ugly? Need link how prettier do this :smile:

Reply:
`?:` isn't valid JS sytax, but [this](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEASeBnGACAvNgMwgmwH5sYAnAVzmyWwCMBDSgCgOqjBgEtpsbAJTZgAHSjYp2VERL4xIFpUUBuCdOwB6LRThYJAXyGqQAGhAQADn2gZkoVpQgB3AAqsE9lMwBuEXgATcxBmLGQCZgAbDDgLRkpmMABrOBgAZSsk3igAc2QqWgtAiDAI6NiLHNjKGDdE3IBbZnKYuJAAKwwADwAhRJS09OZGuAAZHLhWystqGCs5gCZp9qzKGuQlZkYATyjoEKtKHJgAdSCYAAtkAA4ABgsjiFjTxKtNo-04Sl8pi0o4ABHai8AH1ZhNFpISJtCyxRq8Ao0doYHK5KJwACK1Ag8BWFhg23OgSuyEWBMSvCiaIAwhBGs1NlBoH8QNRYgAVbbeGGxQyGIA) is the closest I think - I can look to see why `foo` isn't on the 1st line

Reply:
updated - let me know what you think. Basically stripped out some extra logic we had for the "null if" case, I _think_ this looks better

Reply:
:ok_hand: Let's go with that!